.github/workflows/e2e-app-store.yml

```diff
@@ -54,6 +54,7 @@ jobs:
           SENDGRID_EMAIL: ${{ secrets.CI_SENDGRID_EMAIL }}
           TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
           TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+          ALLOWED_HOSTNAMES: '"cal.com","cal.dev","cal-staging.com","cal.community","cal.local:3000","localhost:3000"'
```

Comment:
@hariombalhara These were sorted alphabetically. Can we maintain that?

Reply:
@hariombalhara Since we now have these variables spread across the files, let's add them to variables. Can you change this line of code to ${{ vars.ALLOWED_HOSTNAMES }}? I'll update GitHub

Reply:
Sure. Making the change.